### ISSUE TITLE
Find data and paper backing current fish background formulae

### ISSUE THREAD

**Issue by erolm-a** [resistance-modelling/slim#42]
The formula (sigmoid on the number of lice per fish) is definitely widespread (e.g. see [this](https://www.researchgate.net/publication/349373813_Modeling_salmon_lice_effects_on_sea_trout_population_dynamics_using_an_individual-based_approach)  ) but the exact coefficients seem to come from nowhere.

**Comment by erolm-a** [comment]
The mystery has been cracked!

The [aforementioned](https://github.com/resistance-modelling/slim/blob/b7314a33bcce7b7d8c179294cc71d4b21eba2bf7/refactor/src/Cage.py#L1072) sigmoid represents the average development time of a single salmon from the moment it is inserted in the cage, and assumes there is a cap development size of 10 kg or 10000 grams after 1500 days or so.

The parameters can be determined as follows: from the [FAO website](http://www.fao.org/fishery/affris/species-profiles/atlantic-salmon/growth/en/) on salmon growth open [figure 5](http://www.fao.org/fileadmin/user_upload/affris/img/fig5.jpg). If we require a minimum of ~80 grams in order for a smolt to be inserted into a cage (as mentioned in the paragraph) and that after one year the average weight is 4.5 kg, while after 3 years the maximum weight reachable is, thus, 10 kg.

Assuming that all fish have comparably the same age, the sigmoid should be a reasonable option. What is currently missing, however, is good documentation (and proper nomenclature).

**Comment by anthonyohare** [comment]
Excellent work
________________________________

**Comment by erolm-a** [comment]
Update on this old issue:

The original question asked for the weight sigmoid coefficients but - in my opinion - it also covered the source of fish background mortality and lice-induced mortality.

As for background mortality we know the coefficient has been calculated by looking at a Scottish government survey, and from the result one would subtract the lice-induced mortality to just get the natural one.

The lice-induced mortality so far was implemented according to a sigmoid function whose coefficients were unknown. To this day I am still not sure where they come from but I decided to switch to the model suggested in [Vollset (2019)](https://doi.org/10.1038/s41598-019-53871-2).
